ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -110,7 +129,9 @@ sub Configure
     my $ok         = 1;
 
     if ($ssh_config->{daemon}) {
-        if ($self->handle_config_file(SSHD_CONFIG, 0600, $ssh_config->{daemon}, \&valid_sshd_file))
+        if ($self->handle_config_file($ssh_config->{daemon}->{config_path}
+                                          || DEFAULT_SSHD_CONFIG,
+                                      0600, $ssh_config->{daemon}, \&valid_sshd_file))
         {
             CAF::Process->new([qw(/sbin/service sshd condrestart)], log => $self)->run();
```

Comment:
actually, `CAF::Service` has no `condrestart`, maybe we should add one so we can do `CAF::Service->new(['sshd'], log => $self)->condrestart()` here and worry about solaris/sysvinit/systemd in `CAF`. @ned21 do you remember why we didn't add `condrestart` to `CAF::Service`?

and it looks like `condrestart` is an undocumented alias for `try-restart` in systemd. so this might work on EL7 afterall...